src/styles/blocks/card.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  width: $main-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__imac-image {
+    height: 134px;
+    width: 160px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__content {
+    font-family: Roboto, sans-serif;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin-top: 0;
+    margin-bottom: 4px;
```

Comment:
The values for font-size and line-height should also be extracted into variables. This will make it easier to maintain consistent typography throughout the application.